config/airflow.cfg

```diff
@@ -30,27 +30,30 @@ logging_config_class =
 # Log format
 # Colour the logs when the controlling terminal is a TTY.
 colored_console_log = True
-colored_log_format = [%%(blue)s%%(asctime)s%%(reset)s] {{%%(blue)s%%(filename)s:%%(reset)s%%(lineno)d}} %%(log_color)s%%(levelname)s%%(reset)s - %%(log_color)s%%(message)s%%(reset)s
+colored_log_format = [%%(blue)s%%(asctime)s%%(reset)s] {%%(blue)s%%(filename)s:%%(reset)s%%(lineno)d} %%(log_color)s%%(levelname)s%%(reset)s - %%(log_color)s%%(message)s%%(reset)s
```

Comment:
could you point out to the changelog that mentions this update?
i looked up on https://github.com/apache/airflow/blob/1.10.5/airflow/config_templates/default_airflow.cfg#L64

there is no change on there

Reply:
This change is correct. In fact, what is currently in master seems to be wrong.

The config you're referencing is a jinja2 template, which after being rendered strips out some braces. Here's how you can easily retrieve a rendered version of the file you've mentioned:

```
$ docker run -it --rm -v `pwd`/config:/conf python:3.7 bash
root@91dd867c925b:/# pip install apache-airflow==1.10.5
root@91dd867c925b:/# airflow initdb
root@91dd867c925b:/# cp ~/airflow/airflow.cfg /conf/airflow_original.cfg
```

And here's the diff between that `airflow_original.cfg` and the file in this branch:

```patch
5c5
< dags_folder = /usr/local/airflow/dags
---
> dags_folder = /root/airflow/dags
9c9
< base_log_folder = /usr/local/airflow/logs
---
> base_log_folder = /root/airflow/logs
42c42
< dag_processor_manager_log_location = /usr/local/airflow/logs/dag_processor_manager/dag_processor_manager.log
---
> dag_processor_manager_log_location = /root/airflow/logs/dag_processor_manager/dag_processor_manager.log
62c62
< # sql_alchemy_conn = sqlite:////tmp/airflow.db
---
> sql_alchemy_conn = sqlite:////root/airflow/airflow.db
118c118
< plugins_folder = /usr/local/airflow/plugins
---
> plugins_folder = /root/airflow/plugins
121c121
< fernet_key = $FERNET_KEY
---
> fernet_key = 
258c258
< expose_config = True
---
> expose_config = False
372c372
< broker_url = redis://redis:6379/1
---
> broker_url = sqla+mysql://airflow:airflow@localhost:3306/airflow
380c380
< result_backend = db+postgresql://airflow:airflow@postgres/airflow
---
> result_backend = db+mysql://airflow:airflow@localhost:3306/airflow
478c478
< child_process_log_directory = /usr/local/airflow/logs/scheduler
---
> child_process_log_directory = /root/airflow/logs/scheduler
681c681
< # i.e. /usr/local/airflow/dags
---
> # i.e. /root/airflow/dags
829c829
< # Should be supplied in the format: key = value
---
> # Should be supplied in the format: key = value
\ No newline at end of file
```